templates/commands/goldentest/verify.go

```diff
@@ -235,3 +248,28 @@ func hasDiff(diffs []diffmatchpatch.Diff) bool {
 	}
 	return false
 }
+
+func getStdoutDiff(goldenStdoutPath, tempStdoutPath string, dmp *diffmatchpatch.DiffMatchPatch) ([]diffmatchpatch.Diff, error) {
+	goldenStdout, err := os.ReadFile(goldenStdoutPath)
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			goldenStdout = []byte("")
+		} else {
```

Comment:
In Go, the usual style is to avoid unnecessary `else`. So this could be:

```
if err != nil {
  if !errors.Is(err, os.ErrNotExist) {
    return nil, fmt.Errorf(........)
  }
  goldenStdout = []byte("")
}
```

Same with `tempStdout` below.

Reply:
done